src/components/RichTextEditor/plugins/linkPlugin/FloatingLink/useFloatingLinkInsert.ts

```diff
@@ -0,0 +1,41 @@
+import {
+    floatingLinkActions,
+    FloatingLinkProps,
+    getSelectionBoundingClientRect,
+    HTMLPropsAs,
+    useComposedRef,
+    useFloatingLinkSelectors,
+    useVirtualFloatingLink,
+} from '@udecode/plate';
+import { useEffect } from 'react';
+
+export const useFloatingLinkInsert = ({ floatingOptions, ...props }: FloatingLinkProps): HTMLPropsAs<'div'> => {
```

Comment:
Not sure if it's somewhere here. :D Placeholders for Text & link are missing in the link modal atm.

Reply:
and also not sure where to add this comment: 
The link modal has a weird positioning when the window size is too small: 
<img width="383" alt="image" src="https://user-images.githubusercontent.com/42832501/189740116-44fd9d79-83e8-41ea-bdcc-55ce20f57e83.png">

Reply:
Good catch. But this was already in the old version an issue.
Maybe lets handle this in a follow up ticket.

Reply:
fair :D. I also think it's not important atm as we don't have mobile support yet.